tests/e2e/e2e_gov_test.go

```diff
@@ -0,0 +1,296 @@
+package e2e
+
+import (
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"time"
+
+	govv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+)
+
+/*
+SendTokensFromNewGovAccount tests passing a gov proposal that sends tokens on behalf of the gov module to a recipient.
+Test Benchmarks:
+1. Subtest to fund the community pool via the distribution module
+2. Submission, deposit and vote of legacy proposal to fund the gov account from the community pool
+3. Validation that funds have been deposited to gov account
+4. Submission, deposit and vote of message based proposal to send funds from the gov account to a recipient account
+5. Validation that funds have been deposited to recipient account
+*/
+func (s *IntegrationTestSuite) SendTokensFromNewGovAccount() {
+	s.writeGovProposals(s.chainA)
+	chainAAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+	senderAddress, err := s.chainA.validators[0].keyInfo.GetAddress()
+	s.Require().NoError(err)
+	sender := senderAddress.String()
+	// Gov tests may be run in arbitrary order, each test must increment proposalCounter to have the correct proposal id to submit and query
+	proposalCounter++
```

Comment:
Can we save this inside the `IntegrationTestSuite` object so we can avoid global variables?

Reply:
also, if we move the test to run in parallel will create a race condition